src/components/table/table.vue

```diff
@@ -374,6 +375,24 @@
                                 };
                             }
                             this.columnsWidth = columnsWidth;
+                            this.fixedHeader();
+
+                            if (this.$refs.tbody) {
+                                let bodyContentEl = this.$refs.tbody.$el;
+                                let bodyEl = bodyContentEl.parentElement;
+                                let className = bodyEl.className;
+                                bodyEl.className = '';
+                                let bodyContentHeight = bodyContentEl.offsetHeight;
+                                let bodyContentWidth = bodyContentEl.offsetWidth;
+                                bodyEl.className = className;
```

Comment:
Why `bodyEl.className = '';`  and then `bodyEl.className = className;`?

Reply:
This setting is to clear the scroll bar. when table changing small the size of the inside is close to the outside size, it will show both scroll bar of horizontal and vertical.

Reply:
@huanghong1125 thanks! What I wonder is why to reset the `.className`? 
Is that to trigger the browser or some mutation observer?

Reply:
@SergioCrisostomo if not reset the class it will display like this
![_20180317193912](https://user-images.githubusercontent.com/12794817/37554952-00a0a59e-2a1b-11e8-82bf-3bb9f07d336b.png)
reset the class
![_20180317193929](https://user-images.githubusercontent.com/12794817/37554978-420e8e7e-2a1b-11e8-9469-adc72bc9b08f.png)